_posts/neatoskeeto/2017-05-22-matt-turtle-hack.md

```diff
@@ -0,0 +1,37 @@
+--- 
+layout: post
+author: neatoskeeto
+title: "Matt's TurtleHack Inspired by Oz"
+---
+
+
+<iframe src="https://trinket.io/embed/python/c424cb87eb" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+I started out doing a grid along the x and y axes, but then I noticed that the grid lines resembled prison bars. I then decided to go with a prison theme based on my favorite tv show Oz.
+I used guess and check to make the eyes look like they are in the right place. It took me a long time to make the mouth, which was just a bunch of guessing and checking. Making the tears was really difficult. I first used the stamp function to creat arrowhead-shaped tears, but I accidentally created a trail of blue behind it because I forgot to do the penup function. I thought the trail looked better. 
```

Comment:
Nicely done on the mouth!  `for` loops will make this much easier in the future, but you put in the work to understand the pattern you need for curves manually.